opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -365,6 +371,24 @@ def __init__(
         self._resource = resource
         self._status = status
 
+    @property
+    def dropped_attributes(self) -> int:
+        if self._attributes:
+            return self._attributes.dropped
```

Comment:
I might be missing something but how is the `dropped` property populated for each of these?

Reply:
dropped is populated via the `_BoundedDict`